src/styles/blocks/stars.css

```
@@ -0,0 +1,39 @@
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(1) {
```

```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2),
... {
```
Consider using commas to group selectors and avoid repeating code